driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -941,6 +978,10 @@ public void run() {
             return ccm;
         }
 
+        private static boolean isThriftSupported(boolean dse, VersionNumber version) {
+            return dse || version.compareTo(VersionNumber.parse("4.0")) < 0;
```

Comment:
I'm just assuming that all versions of DSE support thrift for now, because I don't know when C* 4.0 will get pulled in DSE.